src/main/kotlin/dev/rosewood/roseskyblock/world/IslandWorldGroup.kt

```diff
@@ -8,6 +8,4 @@ class IslandWorldGroup(
     val startingWorld: IslandWorld,
     val gamemode: GameMode,
     val worlds: List<IslandWorld>
-) {
-
-}
```

Comment:
Put braces back, class still needs functionality added and just doesn't have any yet